API/routes/Usuario/User.ts

```diff
@@ -13,12 +15,26 @@ const getUser = new GetUsersValidate();
 const deleteUser = new DeleteUserValidate();
 const updateUser = new UpdateUserValidate();
 const jwt = new Auth();
+/*
+const apiLimiter = new RateLimit({
+  windowMs: 15*60*1000, // 15 minutes 
+  max: 100,
+  delayMs: 0 // disabled 
+});
+*/
+
+const createAccountLimiter = new RateLimit({
+  windowMs: 60*60*10, // 1 hour window 
+  max: 5, // start blocking after 5 requests 
+  message: "Too many accounts created from this IP, please try again after an hour"
+  
+});
 
 /** 
  *  ROTA DE VALIDAÇÃO DE LOGIN
  * */ 
 
-router.post('/', async (req: any, res: any) => {
+router.post('/', createAccountLimiter ,  async  (req: any, res: any) => {
```

Comment:
createAccountLimiter não deveria estar no metodo de registro de usuário?

todo login que for feito será contabilizado no limiter?